modules/protect.php

```diff
@@ -524,8 +524,14 @@ function kill_login() {
 		do_action( 'jpp_kill_login', $ip );
 		$help_url = 'http://jetpack.me/support/security/';
 
+		$die_string = sprintf( __( 'Your IP (%1$s) has been flagged for potential security violations.  <a href="%2$s">Find out more...</a>', 'jetpack' ), str_replace( 'http://', '', esc_url( 'http://' . $ip ) ), esc_url( $help_url ) );
+
+		if( defined('XMLRPC_REQUEST') && XMLRPC_REQUEST ) {
```

Comment:
Ah, yep...

"Always put spaces after commas, and on both sides of logical, comparison, string and assignment operators."

So many spaces.